lib/main_containers.pm

```diff
@@ -60,6 +60,14 @@ sub load_container_engine_test {
     loadtest('containers/container_engine', run_args => $run_args, name => $run_args->{runtime});
 }
 
+sub load_helm_test {
+    my ($k8s_env) = @_;
+    my $run_args = OpenQA::Test::RunArgs->new();
```

Comment:
do you need to create a new `run_args`? can't we pass the var from `load_host_tests_helm` to this method ?

Reply:
New approach